src/app/components/content/IsaacClozeQuestion.tsx

```diff
@@ -19,13 +19,23 @@ import {v4 as uuid_v4} from "uuid";
 import {Item} from "../elements/markup/portals/InlineDropZones";
 import {buildUseKeyboardSensor} from "../../services/clozeQuestionKeyboardSensor";
 
-const augmentInlineItemWithUniqueReplacementID = (idv: ClozeItemDTO) => idv ? ({...idv, replacementId: `${idv?.id}-${uuid_v4()}`}) : undefined;
+import {isDefined} from "../../services/miscUtils";
+
+const augmentInlineItemWithUniqueReplacementID = (idv: ClozeItemDTO | undefined) => isDefined(idv) ? ({...idv, replacementId: `${idv?.id}-${uuid_v4()}`}) : undefined;
 const augmentNonSelectedItemWithReplacementID = (item: ClozeItemDTO) => ({...item, replacementId: item.id});
-const itemNotInAttempt = (currentAttempt: ItemChoiceDTO) => (i: ClozeItemDTO) => !currentAttempt.items?.map(si => si?.id).includes(i.id);
+const itemNotNullAndNotInAttempt = (currentAttempt: {items?: (ItemDTO | undefined)[]}) => (i: ClozeItemDTO | undefined) => i ? !currentAttempt.items?.map(si => si?.id).includes(i.id) : false;
+
+const NULL_CLOZE_ITEM_ID = "NULL_CLOZE_ITEM" as const;
+const NULL_CLOZE_ITEM: ItemDTO = {
+    type: "itemChoice",
```

Comment:
```suggestion
    type: "item",
```

This is incorrect! And I didn't review this before merging it because it was approved 🤦

Reply:
I'll fix it now!

Reply:
It turns out that, since both types inherit from `Content` and since we don't explicitly check the type of the submitted items (I think maybe we should?!), it does work whatever value this has. But it's fixed now anyway . . .

Reply:
Woops, I'd never have spotted that…